### PR TITLE
Update tutorials, closes MISC-212

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: content/tutorials/_includes/upload_manual.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+Let's get back to Gatling Enterprise. 
+To navigate to the artifacts page, click on the **Artifacts** on the left navigation bar.
 
 {{< img src="artifact_navigation.png" alt="Sidenav navigation" >}}
 
-Click on the create button in order to create an artifact in FrontLine. Choose the name you want, then click save.
+Click on the create button in order to create an artifact. 
+Choose the name you want, then click save.
 
 {{< img src="create_artifact_button.png" alt="Create an artifact" >}}
```
Comment (guilgaly) [inline content/tutorials/_includes/upload_manual.md:10]: Line 11: "You now have to upload the artifact you generated earlier. Click on the {{< icon cloud-upload-alt >}} icon."

File: content/tutorials/_includes/upload_manual.md [hunk 2]
```diff
@@ -11,3 +13,5 @@ You now have to upload the artifact you've generated earlier with Maven. Click o
 {{< img src="open_upload.png" alt="Open upload modal" >}}
 
 The artifact you've generated earlier is located here: `{param}`
```
Comment (guilgaly) [inline content/tutorials/_includes/upload_manual.md:15]:
```suggestion
The artifact you generated earlier is located here: `{param}`
```

File: content/tutorials/gradle/index.md [hunk 3]
```diff
@@ -44,23 +45,15 @@ That's it, you've created your first Gatling artifact!
...
-
 {{< include upload_manual.md "frontline-samples/gradle/build/libs/gradle.jar" >}}
 
-Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
-
 {{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}
 
-You now have to click on the Upload button to upload it to FrontLine.
+You now have to click on the Upload button to upload it.
 After a few seconds, the upload will be complete, and your artifact will be ready to use!
```
Comment (guilgaly) [inline content/tutorials/gradle/index.md:53]:
```suggestion
Click on the Upload button. After a few seconds, the upload will complete, and your artifact will be ready to use!
```

File: content/tutorials/maven/index.md [hunk 4]
```diff
@@ -1,26 +1,27 @@
...
+
+There are multiple ways to create a Gatling artifact, which will be used by Gatling Enterprise to launch a run.
 You can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
 
-In this tutorial, we'll use the Maven build tool, so make sure you have Maven configured.
-You'll use Maven from the terminal, but you can also do it easily with an IDE like IntelliJ.
+In this tutorial, we'll use the **Maven** build tool, so make sure you have Maven configured.
 
-## Download the sample Gatling simulation
+You'll use Maven from the terminal, but you can also do it easily with an IDE like IntelliJ.
```
Comment (guilgaly) [inline content/tutorials/maven/index.md:19]:
```suggestion
In this tutorial, we will use Maven from the terminal, but you can also easily run Maven commands from an IDE such as IntelliJ IDEA.
```

File: content/tutorials/maven/index.md [hunk 5]
```diff
@@ -37,35 +38,29 @@ Open a terminal in the maven folder and execute the following command to create
...
 
-Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
 
-{{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}
+{{< img src="choose-artifact.png" alt="Choose the generated artifact" >}}
 
-You now have to click on the Upload button to upload it to FrontLine.
+You now have to click on the Upload button to upload it.
 After a few seconds, the upload will be complete, and your artifact will be ready to use!
```
Comment (guilgaly) [inline content/tutorials/maven/index.md:55]:
```suggestion
Click on the Upload button. After a few seconds, the upload will complete, and your artifact will be ready to use
```

File: content/tutorials/sbt/index.md [hunk 6]
```diff
@@ -42,26 +44,20 @@ sbt test:assembly
...
 {{< include upload_manual.md "frontline-samples/sbt/target/sbt-frontline-1.0.0.jar" >}}
 
-Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
 
 {{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}
 
-You now have to click on the Upload button to upload it to FrontLine.
+You now have to click on the Upload button to upload it.
 After a few seconds, the upload will be complete, and your artifact will be ready to use!
```
Comment (guilgaly) [inline content/tutorials/sbt/index.md:56]:
```suggestion
Click on the Upload button. After a few seconds, the upload will complete, and your artifact will be ready to use
```

File: content/tutorials/sbt/index.md [hunk 7]
```diff
@@ -1,25 +1,27 @@
...
+## Download the sample Gatling simulation
 
-In this tutorial, we'll use the sbt build tool, so make sure you have sbt configured and Java installed.
-You'll use sbt from the terminal, but you can also do it easily with an IDE like IntelliJ.
+There are multiple ways to create a Gatling artifact, which will be used by Gatling Enterprise to launch a run.
+You can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
+
+In this tutorial, we'll use the **SBT** build tool, so make sure you have SBT configured.
+
+You'll use SBT from the terminal, but you can also do it easily with an IDE like IntelliJ.
```
Comment (guilgaly) [inline content/tutorials/sbt/index.md:19]:
```suggestion
In this tutorial, we will use SBT from the terminal, but you can also easily run SBT commands from an IDE such as IntelliJ IDEA.
```

File: content/tutorials/sbt/index.md [hunk 8]
```diff
@@ -1,25 +1,27 @@
...
 {{< include introduction.md >}}
 
-There are multiple ways to create a Gatling artifact, which will be used by FrontLine to launch a run.
-You can use either a build tool (Maven, sbt, Gradle) or the Gatling bundle.
+## Download the sample Gatling simulation
 
-In this tutorial, we'll use the sbt build tool, so make sure you have sbt configured and Java installed.
-You'll use sbt from the terminal, but you can also do it easily with an IDE like IntelliJ.
+There are multiple ways to create a Gatling artifact, which will be used by Gatling Enterprise to launch a run.
+You can use either a build tool (Maven, Sbt, Gradle) or the Gatling bundle.
```
Comment (guilgaly) [inline content/tutorials/sbt/index.md:15]:
```suggestion
You can use either a build tool (Maven, SBT, Gradle) or the Gatling bundle.
```
Notes:
- On the official SBT website, it's spelled "sbt" (all lowercase), but I think in most of the docs we use "SBT" (all uppercase). Let's at least be consistent.
- Same thing for the other tutorials.

File: content/tutorials/gradle/index.md [hunk 9]
```diff
@@ -44,23 +45,15 @@ That's it, you've created your first Gatling artifact!
 
 ## Upload the artifact to FrontLine
```
Comment (guilgaly) [inline content/tutorials/gradle/index.md:46]:
```suggestion
## Upload the artifact to Gatling Enterprise
```

File: content/tutorials/gradle/index.md [hunk 10]
```diff
@@ -44,23 +45,15 @@ That's it, you've created your first Gatling artifact!
...
+You now have to click on the Upload button to upload it.
 After a few seconds, the upload will be complete, and your artifact will be ready to use!
 
 {{< img src="upload.png" alt="Start the upload" >}}
 
-### Option 2: API Upload
-
-{{< include upload_api.md "frontline-samples/gradle/build/libs/gradle.jar" >}}
-
 ## Create a FrontLine simulation
```
Comment (guilgaly) [inline content/tutorials/gradle/index.md:57]:
```suggestion
## Create a Gatling Enterprise simulation
```

File: content/tutorials/_includes/resources.md [hunk 11]
```diff
@@ -1,5 +1,5 @@
 You have successfully started load testing, you can read our documentation if you need more information about our product:
 
-- [FrontLine documentation]({{< ref "../../" >}})
+- [Gatling Enterprise documentation]({{< ref "../../" >}})
 - [Documentation about the creation of an artifact]({{< ref "../../reference/user/reports" >}})
 - [Documentation about how to use the FrontLine reports]({{< ref "../../reference/user/reports" >}})
```
Comment (guilgaly) [inline content/tutorials/_includes/resources.md:5]:
```suggestion
- [Documentation about how to use the Gatling Enterprise reports]({{< ref "../../reference/user/reports" >}})
```

File: content/tutorials/_includes/run.md [hunk 12]
```diff
@@ -1,12 +1,13 @@
...
 
 You can see the status of the simulation in the "Status" column, and the logs by clicking on the {{< icon file-alt >}} button.
 
 {{< img src="simulation_ongoing.png" alt="Simulation status" >}}
 
-You also have access to advanced metrics about your run, even while the simulation is still running. Click on the {{< icon chart-area >}} button to see the reports.
+You also have access to advanced metrics about your run, even while the simulation is still running. 
+Click on the {{< icon chart-area >}} button to see the reports.
 
 {{< img src="reports.png" alt="FrontLine Reports" >}}
```
Comment (guilgaly) [inline content/tutorials/_includes/run.md:12]:
```suggestion
{{< img src="reports.png" alt="Gatling Enterprise Reports" >}}
```

File: content/tutorials/bundle/index.md [hunk 13]
```diff
@@ -32,8 +34,6 @@ Run the `artifact.bat` or `artifact.sh` file, Gatling will start creating your f
 
 ## Upload the artifact to FrontLine
```
Comment (guilgaly) [inline content/tutorials/bundle/index.md:35]:
```suggestion
## Upload the artifact to Gatling Enterprise
```

File: content/tutorials/bundle/index.md [hunk 14]
```diff
@@ -32,8 +34,6 @@ Run the `artifact.bat` or `artifact.sh` file, Gatling will start creating your f
...
 
-### Option 1: Manual Upload
-
 {{< include upload_manual.md "target/artifact.jar" >}}
 
 Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
```
Comment (guilgaly) [inline content/tutorials/bundle/index.md:39]:
```suggestion
Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
```

File: content/tutorials/sbt/index.md [hunk 15]
```diff
@@ -1,25 +1,27 @@
...
+
+You'll use SBT from the terminal, but you can also do it easily with an IDE like IntelliJ.
 
-## Download the sample Gatling simulation
 
 {{< include download_samples.md >}}
 
-Extract the archive, then navigate to the sbt folder.
+Extract the archive, then navigate to the **sbt** folder.
 This folder contains a basic Gatling simulation which can be used with FrontLine.
```
Comment (guilgaly) [inline content/tutorials/sbt/index.md:25]:
```suggestion
This folder contains a basic Gatling simulation which can be used with Gatling Enterprise.
```

**Review by guilgaly — COMMENTED** [review]
In `_includes/simulation_2.md`: "Frontline => "Gatling Enterprise""

File: content/tutorials/_includes/introduction.md [hunk 16]
```diff
@@ -2,11 +2,11 @@
 
 In this tutorial, we'll describe every steps to run your first Simulation with Gatling Enterprise.
 
-It all begins on [cloud.gatling.io](https://cloud.gatling.io), where a GitHub account is needed in order to connect.
+It all begins on [cloud.gatling.io](https://cloud.gatling.io), where you will need a GitHub account is needed in order to sign up.
```
Comment (Isammoc) [inline content/tutorials/_includes/introduction.md:5]:
```suggestion
It all begins on [cloud.gatling.io](https://cloud.gatling.io), where you will need a GitHub account in order to sign up.
```

**Review by Isammoc — APPROVED** [review]

File: content/tutorials/_includes/introduction.md [hunk 17]
```diff
@@ -1,6 +1,20 @@
-In this tutorial, we'll describe every step you need to take in order to have a working basic configuration on FrontLine.
+## Introduction
+
+In this tutorial, we'll describe every steps to run your first Simulation with Gatling Enterprise.
```
Comment (ccedric) [inline content/tutorials/_includes/introduction.md:3]: step without an s

Comment (ccedric) [reply content/tutorials/_includes/introduction.md:3]: I feel like a word is missing, maybe: every step needed to run ?

**Pull request merged**